packages/core/lib/settings/asset_type.dart

```diff
@@ -4,7 +4,9 @@ import 'package:path/path.dart' as p;
 
 /// https://github.com/dart-lang/mime/blob/master/lib/src/default_extension_map.dart
 class AssetType {
-  AssetType(this.path);
+  AssetType(this.path, String rootPath) : absolutePath = p.join(rootPath, path);
+
+  final String absolutePath;
```

Comment:
I don't want to widen the influence range because it is only used in LottieIntegration#isSupport.